ElasticaAdmin/GenerateForm/ElasticaListStrategy.php

```diff
@@ -32,13 +32,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     }
 
     /**
-     * Get the default configuration for the block
-     *
      * @return array
      */
     public function getDefaultConfiguration()
     {
-        return array('searchLimit' => 10);
+        return array(
```

Comment:
@alavieille : we should merge this default with the parent one as we don't know if they evolves in the future